modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -24,8 +27,9 @@ rules:
   resources: ["secrets"]
   verbs: ["*"]
 ----
-. Click *Create* to add the cluster role.  
-. Now create the cluster role binding. In the web console, select *User Management* -> *Role Bindings* -> *Create Binding*.
+
+. Click *Create* to add the cluster role.
+. Now create the cluster role binding. In the main menu, again select *User Management* -> *Role Bindings* -> *Create Binding*.
```

Comment:
What is the `main menu`? That phrase is not currently used in the gitops docs.

Reply:
Hi @kalexand-rh , 
![image](https://github.com/openshift/openshift-docs/assets/99152915/da4d135c-4c88-4f1f-9877-fb4e078377cd)